dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+
+AFRICA:
+  EAST:
+    'BI': [Burundi, Burúndi, Burundian*, Burundiense*, Burundinês*, Bujumbura]
+    'DJ': [Djibouti, Jibuti, Djiboutian*]
+    'ER': [Eritreia, Eritreu*, Eritréia*, Eritreense*, Asmara]
+    'ET': [Etiópia, Etíope*, Etiopian*, Adis Abeba]
+    'KE': [Quénia, Quênia, Quenian*, Nairóbi]
+    'KM': [Comores, Comoriense*, Comorian*, Moroni]
+    'MG': [Madagáscar, Madagascar, Madagascarense*, Malgaxe*, Antananarivo]
+    'MU': [Maurícia, Maurício, Maurician*, Porto Luís]
+    'MW': [Malawi, Maláui, Malaui, Malauian*, Malawian*, Malavian*, Malawiense*, Malauiense*, Malavita*, Lilongué, Lilôngue, Lilóngue, Lilongue, Lilongwe]
+    'MZ': [Moçambique, Moçambican*, Maputo]
+    'RE': [Reunião, Ilha da Reunião, Reunionense*, Reunionês*, Reuniones*]
+    'RW': [Ruanda, Ruandês*, Quigali]
+    'SC': [Seicheles, Seychelles, Seychellense*, Seichelense*]
+    'SO': [Somália, Somali*, Mogadíscio, Mogadixo]
+    'TZ': [Tanzânia, Tanzania, Tanzanian*, Dodoma, Dar es Salaam, Dar es Salã]
+    'UG': [Uganda, Ugandense*, Ugandês*, Campala]
+    'YT': [Mayotte, Maiote, Maiotense*, Mamudzu]
+    'ZM': [Zâmbia, Zambian*, Lusaca]
+    'ZW': [Zimbabwe, Zimbábue, Zimbabué, Zimbaué, Zimbabuean*, Zimbabwean*, Zimbabuan*, Zimbabwan*, Zimbabuense*, Harare]
+
+  MIDDLE:
+    'AO': [Angola, Angolan*, Angolense*, Luanda]
+    'CD': [República Democrática do Congo, RDC, Congo, Congo-Quinxassa, Congo-Quinxasa, Congolês*, Congolense*, Conguês*, Quinxassa, Quinxasa, Kinshasa]
+    'CF': [República Centro-Africana, República da África-Central, Centro-African*, Bangui]
+    'CG': [República do Congo, Congo-Brazavile, Congo-Brazzaville , Congolês*, Conguês*, Conguito*, Congolense*, Brazavile, Brazzaville]
+    'CM': [Camarões, Cameronense*, Cameronês*, Iaundê, Iaundé]
+    'GA': [Gabão, República Gabonesa, Gabonense*, Gabonês*, Libreville, Librevile]
+    'GQ': [Guiné Equatorial, Guineense*, Guinéu-Equatorian*, Equato-Guineense*, Malabo]
+    'ST': [São Tomé e Príncipe, Santomense*, São-Tomense*, São Tomé]
+    'TD': [Chade, Tchade, Chadian*, Chadiense*, Jamena, N'Djamena]
+
+  NORTH:
+    'DZ': [Argélia, República Argelina Democrática e Popular, Argelin*, Argel]
+    'EG': [Egito, Egípci*, Egipcian*, Egipcíac*, Cairo]
+    'EH': [Saara Ocidental, Sara Ocidental, Sáara Ocidental, Saaráui*, Saarian*, Sarian*, El Aiune, Laiune]
+    'LY': [Líbia, Líbio*, Líbic*, Trípoli]
+    'MA': [Marrocos, Marroquin*, Rabat, Rabate]
+    'SD': [Sudão, Sudanês*, Cartum]
+    'SS': [Sudão do Sul, Sudão Meridional, Sul-Sudanês*, Juba]
+    'TN': [Tunísia, República Tunisina, Tunisian*, Tunisin*, Tunes, Túnis]
+
+  SOUTH:
+    'BW': [Botswana, Botsuana, Bechuan*, Botsuanense*, Botsuan*, Botsuanês*, Gaborone]
+    'LS': [Lesoto, Lessoto, Lesotian*, Lestotense*, Lesoto*, Maseru]
+    'NA': [Namíbia, Namibian*, Vinduque]
+    'SZ': [Essuatíni, Essuatinian*, Lobamba, Mebabane, Mbabane]
```

Comment:
For consistency, probably eSwatini should be added to all languages, since the country the changed official name. I see in the it dictionary, but not in the other languages.

Reply:
Added